src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -0,0 +1,23 @@
+<%@ page import="java.time.LocalDate" %>
+<%@ page import="java.time.LocalTime" %>
+<%!
+    public LocalDate getDate() {
+        return LocalDate.now();
+    }
+
+    public String getTime() {
+        LocalTime now = LocalTime.now();
+       return now.getHour() + ":" + now.getMinute() + ":" + now.getSecond();
+    }
+%>
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>index page</title>
+</head>
+<body>
+<h1>
+    <div style="text-align: center;">"<%=getDate()%>" : "<%=getTime()%>"</div>
```

Comment:
According to the task, you have to write "Information written to our resource" message, but I`m agree, that this way it looks cooler 😄

Reply:
ty)